doc/1/essentials/error-handling/index.md

```diff
@@ -5,3 +5,13 @@ title: Error Handling
 description: How to handle errors with the SDK
 order: 100
 ---
+
+# Error handling
+
+All SDK methods can throw subclasses of [KuzzleException](/sdk/csharp/1/exceptions/kuzzle-exception) in case of failure.
```

Comment:
```suggestion
All SDK methods can throw exception inheriting from [KuzzleException](/sdk/csharp/1/exceptions/kuzzle-exception) in case of failure.
```